### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by IvanFadieiev**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+- Docker
+- Postman
+
+### Functionalities
+
+As an unauthorized user you can:
+ - Register
+ - Login
+
+As an authorized user you can:
```
Comment (andrii-hoienko) [inline Readme.md:28]: rewrite it mentioning controllers functionality

File: Readme.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
+
+![Banner](banner.jpg)
```
Comment (andrii-hoienko) [inline Readme.md:2]: Please add loom video

File: Readme.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+### Introduction
+
+BookStore application provides an easy and effective way to purchase books from online book store.
+
+### Technologies 
+
+- SpringBoot
```
Comment (andrii-hoienko) [inline Readme.md:10]: Add details about purpose of tools you used

File: Readme.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+As an authorized administrator you can:
+- Create a new book
+- Update/Delete existing book
+- Create a new category
+- Update/Delete category
+- Change order status
+
+
+### Installation
```
Comment (andrii-hoienko) [inline Readme.md:50]: Add guide for alternative way to run app

Comment (andrii-hoienko) [inline Readme.md:50]: What tools required to run app?

File: Readme.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+        ```
+3. Finally, use Docker to run this app in container
+        ```
+        docker-compose build
+        ```
+        ```
+        docker-compose up
+        ```
+        
+### Postman requests collection
```
Comment (andrii-hoienko) [inline Readme.md:68]: Tell something about Contributing

File: Readme.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+- Hibernate
+- MySQL
+- Maven
+- Git
+- Liquibase
+- Swagger
+- Docker
+- Postman
+
+### Functionalities
```
Comment (andrii-hoienko) [inline Readme.md:22]: mention security and validations rules

**Review by andrii-hoienko — CHANGES_REQUESTED** [review]
GJ! But still, some improvements required

File: Readme.md (hunk 4 above)
Comment (andrii-hoienko) [reply Readme.md]: It is not enough to just have a docker

Comment (andrii-hoienko) [reply Readme.md]: Not resolved, by an alternative way I mean start app without docker (using java only)

**Review by andrii-hoienko — CHANGES_REQUESTED** [review]
Some comments are not resolved

**Review by vovasalyha — APPROVED** [review]

**Pull request merged**